installer/pkg/common/storage.go

```diff
@@ -33,13 +39,27 @@ func StorageConfig(context *RenderContext) storageconfig.StorageConfig {
 			GCloudConfig: storageconfig.GCPConfig{
 				Region:             context.Config.Metadata.Region,
 				Project:            context.Config.ObjectStorage.CloudStorage.Project,
-				CredentialsFile:    "/mnt/secrets/gcp-storage/service-account.json",
+				CredentialsFile:    getStoragePath("service-account.json"),
 				ParallelUpload:     6,
 				MaximumBackupCount: 3,
 			},
 		}
 	}
 
+	if context.Config.ObjectStorage.S3 != nil {
+		res = &storageconfig.StorageConfig{
+			Kind: storageconfig.MinIOStorage,
+			MinIOConfig: storageconfig.MinIOConfig{
+				Endpoint:            context.Config.ObjectStorage.S3.Endpoint,
+				AccessKeyIdFile:     getStoragePath("username"),
```

Comment:
Why is this file called "username" instead of `accessKeyId`?
Conversely for the `SecretAccessKey`?

Reply:
Minio and the S3 docs I was looking at called it user/pass and I didn't think of a better name. Will change to `accessKeyId` and `secretAccessKey`